src/pages/hegel/reference/becoming/development.mdx

```diff
@@ -0,0 +1,562 @@
+> Being, is inadequate and leads to contradictions which prove its falsity. The
+> only truth of the two is a category which expresses the relation of the two
+> (McTaggart 1910, 17).
+
+There is no contradiction in the union of `being` and `nothing`, at least not in
+the manner seen in the development of `becoming` concerning their exact identity
+and pure difference.
+
+Now, the issues McTaggart sees in the term "becoming", are specifically:
+
+1. The term "becoming" evokes the idea of a process of change. A change of
+   something to something else.
+1. Change, in turn, involves the existence of some permanent element in what
+   changes. An element that does not change. Put differently, change requires
+   some stable form to contrast the contents that were changed.
+1. It is impossible in the categories of `being` and `nothing`, since there is
+   no room for distinction between a changing and an unchanging layer of reality
+   as the categories are simply undifferentiated.
+1. Therefore, if something should be capable of change, it must be analyzable
+   into two elements, one which does not change. "This is impossible under the
+   categories of Quality", McTaggart claims (McTaggart 1910, 17).
+
+McTaggart considers the scenario that `becoming` may not be a fully developed
+sense of change, but change in a more rudimentary form. Rather than change of A
+into B, it is the disappearance of A and the appearance of B. But McTaggart
+thinks that even in the latter case, when carefully analyzed, would "involve the
+presence of some element which persisted unchanged in connection first with A
+and then B", and that would just be change proper (McTaggart 1910, 19).
+
+Furthermore, McTaggart contents that the essence of the new category
+(`existence`) is in the necessary implication of `being` and `nothing` and "not
+in any change taking place between them" (McTaggart 1910, 19).
+
+Reviewing McTaggart commentary on `becoming`, it appears to deal more possible
+semantic implications rather than with the technicalities of the argument, which
+problematizes his reading. Here are a few points to consider.
+
+1. Firstly, nowhere does Hegel employ "change" to describe the movement of
+   `becoming`; rather, he carefully combines ideas of "vanishing". So while
+   McTaggart is right that change requires some permanent element, and therefore
+   does not map on to the logic of `becoming`, it was never asked for by the
+   argument of the _Logic_. Instead, it is better to think of `becoming` not as
+   any species of change but simply as _becoming_, a unique logical movement.
+1. Secondly, McTaggart's understanding that `existence` is merely the necessary
+   implication of `being` and `nothing`, omits the important point that these
+   can no longer be thought in their purity (see
+   [Houlgate's commentary on this](#houlgate)).
+1. Thirdly, misnaming `becoming` with preference for the much more generic
```

Comment:
Though I have to say that there is something very appealing about McTaggart's choice.

Reply:
Fair enough, but could say more exactly what it is? 😀

Reply:
It's an unnecessarily convoluted point about proper nouns and descriptive nouns, and how the former are possibly more troubling than the latter for the SL. But this is a conversation for another thread